ruby/rails/security/brakeman/check-before-filter.yaml

```diff
@@ -0,0 +1,27 @@
+rules:
+- id: check-before-filter
+  mode: search
+  patterns:
+  - pattern-either:
+    - pattern: |
+        skip_filter ..., :except => $ARGS
+    - pattern: |
+        skip_before_filter ..., :except => $ARGS
```

Comment:
this is less opinionated than Brakeman on second look - Brakeman uses specific checks